index.js

```diff
@@ -14,13 +13,6 @@ var testUtil = require('./lib/util')
  * @module test-utilities
  */
 
-exports = module.exports = function extendTap(tap) {
-  metrics.extendTap(tap)
-  TestAgent.extendTap(tap)
-
-  return exports
-}
-
```

Comment:
I probably though it was not necessary now that there is just one call, but actually makes sense to have this hiding the details.  I will put it back.